packages/flutter/lib/src/material/menu_anchor.dart

```diff
@@ -871,6 +872,11 @@ class MenuItemButton extends StatefulWidget {
   /// An optional icon to display after the [child] label.
   final Widget? trailingIcon;
 
+  /// Determines if the menuAnchor should be closed after a [MenuItemButton]
+  /// has been pressed.
+  /// Defaults to true.
```

Comment:
Since these have the same description, you should make them into a template so that they stay in sync:

```suggestion
  /// {@template flutter.material.menu_anchor.closeOnActivate}
  /// Determines if the menu will be closed when a [MenuItemButton]
  /// is pressed.
  ///
  /// Defaults to true.
  /// {@endtemplate}
```

Reply:
done